src/performance/navigation.ts

```diff
@@ -0,0 +1,27 @@
+import { Event } from "@clarity-types/data";
+import { NavigationData } from "@clarity-types/performance";
+import encode from "./encode";
+
+export let data: NavigationData = null;
+
+export function reset(): void {
+    data = null;
+}
+
+export function compute(entry: PerformanceNavigationTiming): void {
+    data = {
+        fetchStart: Math.round(entry.fetchStart),
+        connectStart: Math.round(entry.connectStart),
+        connectEnd: Math.round(entry.connectEnd),
+        requestStart: Math.round(entry.requestStart),
+        responseStart: Math.round(entry.responseStart),
+        responseEnd: Math.round(entry.responseEnd),
+        domInteractive: Math.round(entry.domInteractive),
+        domComplete: Math.round(entry.domComplete),
+        loadEventEnd: Math.round(entry.loadEventEnd),
+        size: entry.transferSize,
```

Comment:
Add redirectCount

Reply:
I will add it, thanks.

---
In reply to: [347587052](https://github.com/microsoft/clarity-js/pull/160#discussion_r347587052) [](ancestors = 347587052)